api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/DetermineEntitlementNotificationHandler.java

```diff
@@ -20,13 +21,17 @@
     private final MessageQueueClient messageQueueClient;
 
     public void sendClaimNoLongerEligibleEmail(Claim claim) {
-        MessagePayload messagePayload = buildClaimIsNoLongerEligibleNotificationEmailPayload(claim);
-        messageQueueClient.sendMessage(messagePayload, SEND_EMAIL);
+        if (StringUtils.isNotEmpty(claim.getClaimant().getEmailAddress())) {
```

Comment:
I was thinking, rather than doing this and checking for null/ empty in all places shall we make the type Optional? In that case you can just call .isPresent